docs/faq.asciidoc

```diff
@@ -143,22 +168,59 @@ INFO      "delete_indices" action completed.
 
 You can read more on Curator here: https://www.elastic.co/guide/en/elasticsearch/client/curator/current/index.html
 
-[float]
-[[reduce-payload-size]]
-==== Reduce the payload size
-
-Large payloads coming from agents may result in "503: Request timed out waiting to be processed" messages.
-You can reduce the payload size by decreasing the `max_queue_size` in the agents.
-This will result in agents sending smaller payloads to the APM Server,
-but the requests will be more frequent.
-See the documentation for the {apm-py-ref}/configuration.html#config-max-queue-size[Python] and {apm-node-ref}/agent-api.html#max-queue-size[Node.js] agents for more information.
+If you want to delete documents that match a specific query, e.g. all documents with a given `context.service.name`,
+you can do this by sending following request against your Elasticsearch API:
+["source","sh",subs="attributes"]
+------------------------------------------------------------
+POST /apm-*/_delete_by_query
+{
+  "query": {
+    "bool": {
+      "must": [
+        {
+          "term": {
+            "context.service.name": {
+              "value": "old-service-name"
+            }
+          }
+        }
+      ]
+    }
+  }
+}
+------------------------------------------------------------
 
-[float]
-[[add-apm-server-nodes]]
-==== Add APM Server nodes
+See https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-delete-by-query.html for further information
+on this topic.
 
-When the APM Server cannot process incoming requests quickly enough,
-you will see "503: Request timed out waiting to be processed" messages.
 
-The best way to avoid this problem is to add more processing power to your APM Server cluster.
-You can either migrate your APM Server processes to more powerful machines or add more machines.
+[float]
+[[update-existing-data]]
+=== How to update existing data?
+In case you want to update specific attributes in existing documents, e.g. update `context.service.name` to a new value, 
+you can do this by sending following request against your Elasticsearch API: 
+["source","sh",subs="attributes"]
+------------------------------------------------------------
+POST /apm-*/_update_by_query
+{
+  "query": {
+    "bool": {
+      "must": [
+        {
+          "term": {
+            "context.service.name": {
+              "value": "old-service-name"
+            }
+          }
+        }
+      ]
+    }
+  },
+  "script": {
+    "source": "ctx._source['context.service.name'] = 'new-service-name'",
+    "lang": "painless"
+  }
+}
+------------------------------------------------------------
+See https://www.elastic.co/guide/en/elasticsearch/reference/6.0//docs-update-by-query.html for further information on 
```

Comment:
Use an asciidoc attribute, as recommended earlier. Not sure why you would hard code to 6.0 branch, but there's a way to do that if you need to. It's generally better to point to branches at the same release level unless there's a compelling reason to specify a version.

Reply:
Even if we need to point to a specific branch, I would still recommend that we use the attributes, e.g. `See {ref-60}/docs-update-by-query.html[update by query] for further...`

https://github.com/elastic/docs/blob/3eac44b52d266ec5b7ad8d9d1cc583558cce2e5c/shared/attributes.asciidoc#L2